### PR TITLE
Connecting sample IS to database

### PR DESCRIPTION
**Pull request by skoruba**
I've added new project with IDS4 that is connected to Asp.Net Core Identity and EF - #30

@xmichaelx - Please take a look. :)

Thank you.

### CODE REVIEW

**Comment by skoruba** [comment]
@TomasHubelbauer - what do you think about it? :)

File: src/Skoruba.IdentityServer4.AspNetIdentity/Quickstart/Grants/GrantsController.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            await _interaction.RevokeUserConsentAsync(clientId);
+            return RedirectToAction("Index");
+        }
+
+        private async Task<GrantsViewModel> BuildViewModelAsync()
+        {
+            var grants = await _interaction.GetAllUserConsentsAsync();
+
+            var list = new List<GrantViewModel>();
+            foreach(var grant in grants)
```
Comment (TomasHubelbauer) [inline src/Skoruba.IdentityServer4.AspNetIdentity/Quickstart/Grants/GrantsController.cs]: You're missing a space after `foreach` if you format code that way.

File: src/Skoruba.IdentityServer4.AspNetIdentity/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                // this adds the operational data from DB (codes, tokens, consents)
+                .AddOperationalStore(options =>
+                {
+                    options.ConfigureDbContext = b =>
+                        b.UseSqlServer(connectionString,
+                            sql => sql.MigrationsAssembly(migrationsAssembly));
+
+                    // this enables automatic token cleanup. this is optional.
+                    options.EnableTokenCleanup = true;
+                    // options.TokenCleanupInterval = 15; // frequency in seconds to cleanup stale grants. 15 is useful during debugging
```
Comment (TomasHubelbauer) [inline src/Skoruba.IdentityServer4.AspNetIdentity/Startup.cs]: Should this be here if commented out? Maybe `#if DEBUG`?

**Comment by skoruba** [comment]
Great tip, I agree. :)

**Comment by TomasHubelbauer** [comment]
Ah, nevermind, so in this case when I collapsed all the `wwwroot` stuff, the actual PR was pretty manageable in size so I reviewed in one sweep. Left a few nit comments but in principle it's approved from me. 👍

File: src/Skoruba.IdentityServer4.AspNetIdentity/Quickstart/Grants/GrantsController.cs (hunk 1 above)
Comment (skoruba) [reply src/Skoruba.IdentityServer4.AspNetIdentity/Quickstart/Grants/GrantsController.cs]: DONE. :)

File: src/Skoruba.IdentityServer4.AspNetIdentity/Startup.cs (hunk 2 above)
Comment (skoruba) [reply src/Skoruba.IdentityServer4.AspNetIdentity/Startup.cs]: DONE. :)

**Pull request merged**